doc/Pacemaker_Explained/en-US/Ch-Resources.txt

```diff
@@ -637,8 +639,10 @@ indexterm:[Action,Property,on-fail]
 
 |enabled
 |TRUE
-|If +false+, the operation is treated as if it does not exist. Allowed
- values: +true+, +false+
+|If +false+, the operation is treated as if it is not defined at all.
+ That does not mean the associated action cannot occur, recurring
+ actions are implicitly suppressed, though.
+ Allowed values: +true+, +false+.
```

Comment:
I'd prefer something like:

If +false+, ignore this operation definition. This is typically used to pause a particular recurring monitor operation; it does not suppress all actions of the given type.